salt/modules/ini_manage.py

```diff
@@ -50,6 +50,10 @@ def set_option(file_name, sections=None):
         If the Ini does not contain sections the keys and values represent the
         options
 
+    separator : =
+        A character used to separate keys and values. Standard ini files use
+        the "=" character.
+
```

Comment:
@cachedout Done, but I figure it should be properly tested before being merged in. I'll need to build a new salt and test state first, so will try to do that today or tomorrow.

@jfindlay, thanks for the versionadded comment hint. I've added such a line to the commit.